### DISCUSSION TITLE
Welcome to strimio-desktop Discussions!

### DISCUSSION THREAD

**Discussion by strimio** [strimio/strimio-desktop#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dido-b** [comment]
The text of Strimio on Windows 11 is much too small - any way to increase it?
Regards, David

**Reply by spiral2k** [reply]
@dido-b there will be a fix for it in the next update.
you'll be able to increase the font size of the text via settings.

**Comment by dido-b** [comment]
Thanks a lot!

Outlook for Android herunterladen
________________________________

**Reply by spiral2k** [reply]
@dido-b  please download the 3.0.9 update, there is a new section in the settings for font size
appreciate your feedback.

**Comment by dido-b** [comment]
Works nicely, now I can read the menu without glasses 🙂. Text in icons (e.g. countries) still small but readable, no problem.

Thanks!
________________________________
Von: Spiral2k ***@***.***>
Gesendet: Dienstag, 25. Oktober 2022 21:06
An: strimio/strimio-desktop ***@***.***>
Cc: dido-b ***@***.***>; Mention ***@***.***>
Betreff: Re: [strimio/strimio-desktop] Welcome to strimio-desktop Discussions! (Discussion #2)


Thanks a lot! Outlook for Android herunterladen
…
________________________________ From: Spiral2k @.> Sent: Tuesday, October 25, 2022 12:37:40 AM To: strimio/strimio-desktop @.> Cc: dido-b @.>; Mention @.> Subject: Re: [strimio/strimio-desktop] Welcome to strimio-desktop Discussions! (Discussion #2<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fstrimio%2Fstrimio-desktop%2Fdiscussions%2F2&data=05%7C01%7C%7Cdc1933dcb8fb4c5083da08dab6bbfc7e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638023215765010777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=euA%2FCvi8MY%2BF0ZPgqrM6jymkOPEG5WUG1Ao%2FjPkbjXk%3D&reserved=0>) @dido-b<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdido-b&data=05%7C01%7C%7Cdc1933dcb8fb4c5083da08dab6bbfc7e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638023215765010777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nOkK2zQOEn8WAK0gp%2FqsJjKD3Py2PpbpwZDT6Y7oB70%3D&reserved=0>https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdido-b&data=05%7C01%7C%7C0be9186f8295497c01a608dab6105ca3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638022478628335469%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=4HQ9HXMMfcfC6Ee62bxyXsrY4h3v%2FmsIYzyYQmFiUmA%3D&reserved=0<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdido-b&data=05%7C01%7C%7Cdc1933dcb8fb4c5083da08dab6bbfc7e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638023215765010777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nOkK2zQOEn8WAK0gp%2FqsJjKD3Py2PpbpwZDT6Y7oB70%3D&reserved=0> there will be a fix for it in the next update. you'll be able to increase the font size of the text via settings. — Reply to this email directly, view it on GitHubhttps://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fstrimio%2Fstrimio-desktop%2Fdiscussions%2F2%23discussioncomment-3955286&data=05%7C01%7C%7C0be9186f8295497c01a608dab6105ca3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638022478628335469%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2FPtDVPgeLfY0DedvABSBJ6R09iqxz%2FCJip0k6yMHVR4%3D&reserved=0<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fstrimio%2Fstrimio-desktop%2Fdiscussions%2F2%23discussioncomment-3955286&data=05%7C01%7C%7Cdc1933dcb8fb4c5083da08dab6bbfc7e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638023215765010777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=34npywi3HkW87eQ34u0HP%2BXhA4iv3BTUkEzj9Lv0H8s%3D&reserved=0>, or unsubscribehttps://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJ7N5HXQPDZIC2US7CD34N3WE4FTJANCNFSM6AAAAAAQZ2BN6E&data=05%7C01%7C%7C0be9186f8295497c01a608dab6105ca3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638022478628335469%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ipC%2BpgrNWoSKOYcSq7d6k9XhrdgtI225D9rEhtfr%2BNE%3D&reserved=0<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAJ7N5HXQPDZIC2US7CD34N3WE4FTJANCNFSM6AAAAAAQZ2BN6E&data=05%7C01%7C%7Cdc1933dcb8fb4c5083da08dab6bbfc7e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638023215765010777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=aFi7na1SrcNm1FvPPrzzkwPCpWat%2FMn6ULGKtC6VKaI%3D&reserved=0>. You are receiving this because you were mentioned.Message ID: @.***>

@dido-b<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdido-b&data=05%7C01%7C%7Cdc1933dcb8fb4c5083da08dab6bbfc7e%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638023215765010777%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=nOkK2zQOEn8WAK0gp%2FqsJjKD3Py2PpbpwZDT6Y7oB70%3D&reserved=0> please download the 3.0.8 update, there is a new option in the settings, ket me know how it works

**Reply by spiral2k** [reply]
@dido-b It was my pleasure to help, I hope you have a great time!

**Comment by HelenWangyj** [comment]
Do we have an android version? Really like this app!
